lighthouse-plugin-publisher-ads/utils/resource-classification.js

```diff
@@ -197,18 +264,24 @@ function getAbbreviatedUrl(url) {
 }
 
 module.exports = {
-  isBidRelatedRequest,
-  isBidRequest,
   isGoogleAds,
-  isGptAdRequest,
-  hasImpressionPath,
-  isGpt,
   isGptTag,
+  isGptImplTag,
+  isGpt,
+  isGptAdRequest,
+  isGptIframe,
+  isAdSense,
+  isAdSenseTag,
```

Comment:
Maybe put AdSense before GPT, to sort alphabetically

Reply:
Done (but not committed yet)

Reply:
...and now committed.